app/src/main/java/mozilla/lockbox/autofill/FillResponseBuilder.kt

```diff
@@ -4,24 +4,34 @@ import android.annotation.TargetApi
 import android.content.Context
 import android.content.IntentSender
 import android.os.Build
+import android.os.Bundle
 import android.service.autofill.Dataset
 import android.service.autofill.FillResponse
-import android.view.autofill.AutofillId
+import android.service.autofill.SaveInfo
 import android.view.autofill.AutofillValue
 import android.widget.RemoteViews
 import io.reactivex.Observable
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import mozilla.appservices.logins.ServerPassword
 import mozilla.lockbox.R
 import mozilla.lockbox.model.titleFromHostname
+import mozilla.lockbox.support.Constant
 import mozilla.lockbox.support.PublicSuffixSupport
 import mozilla.lockbox.support.filter
 
 @TargetApi(Build.VERSION_CODES.O)
 @ExperimentalCoroutinesApi
 open class FillResponseBuilder(
-    internal val parsedStructure: ParsedStructure
+    internal val parsedStructure: ParsedStructure,
+    private val enableSave: Boolean = true
```

Comment:
This acts as the Feature Flag, correct? Will we change the signature once we remove the Feature Flags? I think I would prefer to just do a check [like we're doing for the delete/edit menu](https://github.com/mozilla-lockwise/lockwise-android/blob/0a272bed74f402f0022e784c9ab25d26ff94087d/app/src/main/java/mozilla/lockbox/presenter/ItemDetailPresenter.kt#L78)